src/sentry/metrics/composite_experimental.py

```diff
@@ -130,3 +130,17 @@ def distribution(
                 unit,
                 stacklevel=stacklevel + 1,
             )
+
+    def event(
+        self,
+        title: str,
+        message: str,
+        alert_type: str | None = None,
+        aggregation_key: str | None = None,
+        source_type_name: str | None = None,
+        priority: str | None = None,
+        instance: str | None = None,
+        tags: Tags | None = None,
+        stacklevel: int = 0,
+    ) -> None:
+        pass
```

Comment:
If this is what we use in getsentry production, this will not work as you will not be producing any event to datadog in prod. 
You'll have to forward the call to `self._primary_banckend`